src/NServiceBus.Transport.RabbitMQ/RabbitMQTransport.cs

```diff
@@ -39,16 +43,21 @@ public RabbitMQTransport(RoutingTopology routingTopology, string connectionStrin
             ArgumentNullException.ThrowIfNull(connectionString);
 
             RoutingTopology = routingTopology.Create();
-            ConnectionConfiguration = ConnectionConfiguration.Create(connectionString);
+            BrokerConnectionConfiguration = ConnectionConfiguration.Create(connectionString);
+
+            ManagementConnectionConfiguration = string.IsNullOrEmpty(managementConnectionString) ?
+                ConnectionConfiguration.ConvertToManagementConnection(BrokerConnectionConfiguration) :
+                ConnectionConfiguration.Create(managementConnectionString, isManagementConnection: true);
         }
 
         /// <summary>
         /// Creates a new instance of the RabbitMQ transport.
         /// </summary>
         /// <param name="routingTopology">The routing topology to use.</param>
         /// <param name="connectionString">The connection string to use when connecting to the broker.</param>
+        /// <param name="managementConnectionString">The connection string to use when connecting to the management API</param>
         /// <param name="enableDelayedDelivery">Should the delayed delivery infrastructure be created by the endpoint</param>
-        public RabbitMQTransport(RoutingTopology routingTopology, string connectionString, bool enableDelayedDelivery)
+        public RabbitMQTransport(RoutingTopology routingTopology, string connectionString, bool enableDelayedDelivery, string managementConnectionString = null)
```

Comment:
We should not use optional parameters in public APIs. They can be hard to version properly. Instead, we should add overloads that take the management connection string.